easybuild/easyconfigs/s/spaceranger/spaceranger-1.1.0-GCC-9.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'Tarball'
+
+name = 'spaceranger'
```

Comment:
I think that the names `SpaceRanger` or `Space-Ranger` are more fitting compared to what is used in their website

Reply:
none of the 3 options really fit the bill because they advertise it as "Space Ranger" in 2 words, that's why I opted for the executable.

but indeed, `SpaceRanger` is more consistent with `CellRanger` so I'll go with that.